### PR TITLE
12.0 mig beesdoo pos

### PR DESCRIPTION
**Pull request by mclaeysb**
Migrated the following modules:

- pos_price_to_weight
- beesdoo_product
- beesdoo_pos

Order receipt functionnality was missing and needs to be added again. I created a task for this titled 'Add order receipt functionnality'.

PS: this my first PR!

### CODE REVIEW

File: beesdoo_pos/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    """,
+
+    'author': "Beescoop - Cellule IT",
+    'website': "https://github.com/beescoop/Obeesdoo",
+
+    # Categories can be used to filter modules in modules listing
+    # Check https://github.com/odoo/odoo/blob/master/openerp/addons/base/module/module_data.xml
+    # for the full list
+    'category': 'Point Of Sale',
+    'version': '9.0.1.1.0',
```
Comment (remytms) [inline beesdoo_pos/__manifest__.py]: Please change version number to 12.0.1.0.0.

File: pos_price_to_weight/data/barcode_rule.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@ Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
 License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 -->
 
-<openerp><data>
+<odoo><data>
```
Comment (remytms) [inline pos_price_to_weight/data/barcode_rule.xml]: For cleaner file, you can remove `<data>` tag.

File: pos_price_to_weight/data/barcode_rule.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@ Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
...
 
     <record id="rule_price_to_weight" model="barcode.rule">
```
Comment (remytms) [inline pos_price_to_weight/data/barcode_rule.xml:9]: Please fix indentation issues. XML files should be indented with `spaces` only or `tabs` only. Not with a mix of them. I prefer `spaces` indentation.

Feel free to fix that when you see a mixing of spaces and tabs in an xml or js file when moving to a new version. :)

File: beesdoo_product/models/beesdoo_product.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
  # -*- coding: utf-8 -*-
-from openerp import models, fields, api
-from openerp.tools.translate import _
-from openerp.exceptions import UserError, ValidationError
+from odoo import models, fields, api
+from odoo.tools.translate import _
+from odoo.exceptions import UserError, ValidationError
 import uuid
```
Comment (remytms) [inline beesdoo_product/models/beesdoo_product.py:6]: Please fix  import ordering. `uuid` is a standard python library and there fore should be imported first. See OCA guidelines : https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#imports

Also you can put a empty line between different source of import, like in the guidelines. :)

**Review by remytms — COMMENTED** [review]
Thanks for your work !

Some little changes. :)

File: pos_price_to_weight/data/barcode_rule.xml (hunk 3 above)
Comment (mclaeysb) [reply pos_price_to_weight/data/barcode_rule.xml:9]: I only find spaces in this file, no tabs? I found mixtures in other `*.xml` files though, I'll fix those in a separate commit.

**Comment by mclaeysb** [comment]
Here are the changes you requested!

**Comment by houssine78** [comment]
@remytms if it ok for you let's merge !

File: beesdoo_product/data/barcode_rule.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+	<data noupdate="1">
+		<record model="barcode.rule" id="beesdoo_product_barcode_rule">
+			<field name="name">Beescoop Product Barcodes</field>
```
Comment (houssine78) [inline beesdoo_product/data/barcode_rule.xml:5]: @elouan-lb you did something recently on Odoo 9 on this matters no ? does this need to be changed accordingly ?

Comment (houssine78) [reply beesdoo_product/data/barcode_rule.xml:5]: @remytms Elouan will show you what have been done on nine. Could you just integrate it on this PR?

File: beesdoo_product/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    """,
+
+    'author': "Beescoop - Cellule IT",
+    'website': "https://github.com/beescoop/Obeesdoo",
+
+    # Categories can be used to filter modules in modules listing
+    # Check https://github.com/odoo/odoo/blob/master/openerp/addons/base/module/module_data.xml
+    # for the full list
+    'category': 'Sales Management',
+    'version': '0.1',
```
Comment (houssine78) [inline beesdoo_product/__manifest__.py:19]: change to 12.0.1.0.0

Comment (houssine78) [reply beesdoo_product/__manifest__.py:19]: maybe add coop it easy in the contributor as part of the works has been made by CIE workers non volunteering to bees coop

**Comment by remytms** [comment]
see #110

**Review by houssine78 — APPROVED** [review]

File: beesdoo_product/data/barcode_rule.xml (hunk 5 above)
Comment (elouan-lb) [reply beesdoo_product/data/barcode_rule.xml:5]: I cherry-picked the commit on migration branch (see e1720dae9106bbb30e91e9d37c454358e0df7ee4)

**Pull request closed without merging**